vaadin-upload-file.html

```diff
@@ -142,14 +142,23 @@
         @apply(--vaadin-upload-file-progress-complete);
       }
 
+      .fadeOut {
+        opacity: 0;
+        transition: opacity 1s ease-out;
+        -ms-transition: opacity 1s ease-out;
+        -moz-transition: opacity 1s ease-out;
+        -webkit-transition: opacity 1s ease-out;
```

Comment:
Transition is supported without prefixes in all the browsers we support, no need to pollute CSS with redundancy: http://caniuse.com/#feat=css-transitions